doc/guides/linux_gsg/linux_gsg.rst

```diff
@@ -145,7 +145,16 @@ On a NUMA machine with two nodes, pages should be allocated explicitly on separa
    For 1GB pages, it is not possible to reserve the hugepage memory after the system has booted.
 
 Prerequisites
-~~~~~~~~~~~~~~
+~~~~~~~~~~~~~
+
+Building CNDP from behind a proxy server
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+If building behind a proxy server the following is required before excuting a git clone or other commands needs to pass thru the proxy server.
```

Comment:
Line is long, I prefer to keep lines, even in documentation <100 chars for consistency with code.

couple of typos

s/excuting/executing
s/thru/through

"or other commands needs to pass thru the proxy server" should be "needing", not "needs"